install.ps1

```diff
@@ -76,7 +76,16 @@ param (
   [switch]$noReboots,
   [switch]$noChecks
 )
-$ErrorActionPreference = "Stop"
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue'
+ 
+# https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.localaccounts/set-localuser
+if ( -not ([Environment]::Is64BitProcess)) {
+    Write-Host "`t[!] It seems like you are using 32 bit powershell. Exiting because some commands do not work correctly in this mode." -ForegroundColor Red
```

Comment:
As powershell x86 is not working at the moment, I think adding this code is a good idea. But on the long-term, is there a reason why we would want to support powershell x86 (we would need to find alternative implementations for the used function(s))? :thinking: @mandiant/vms opinions?